src/main/java/duke/Storage.java

```diff
@@ -0,0 +1,110 @@
+package duke;
```

Comment:
Adding a line break to separate the package statement from the import statements might add clarity! Similar for other classes.

Reply:
Thank you